lib/iron_bank/local_records.rb

```diff
@@ -36,17 +36,17 @@ def save_file
     private
 
     BACKOFF = {
-      max:      10,
```

Comment:
why did we have to get rid of this and the other keys?

Reply:
It's not needed here because now we're passing that value in